### PR TITLE
fix: upgrade containers

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
Updated postgres and the other outdated containers

### Description of PR

What component(s) does this PR affect?

- [x] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -527,13 +527,13 @@ services:
         volumes:
             - ./postgres-data-current:/var/lib/postgresql/data
             - ./other/postgres-entrypoint.sh:/postgres-entrypoint.sh
-            - ./postgres-data-backup/:/tmp/
+            - ./postgres-data-backup/:/postgres-entrypoint.d/
```
Comment (vkotronis) [inline docker-compose.yaml:530]: is this correct for backup data mappings?

File: docker-compose.yaml [hunk 2]
```diff
@@ -510,7 +510,7 @@ services:
         volumes:
             - ./other/rabbitmq/enabled-plugins:/etc/rabbitmq/enabled_plugins
     postgres:
-        image: timescale/timescaledb:1.1.0-pg10
+        image: timescale/timescaledb:2.7.0-pg14
```
Comment (vkotronis) [inline docker-compose.yaml:513]: we should note that this change is not bw-compatible and requires DB to be gone before upgrading

**Comment by vkotronis** [comment]
Please make an empty commit without skipping to force all tests (+ integration) to run with new containers.

File: docker-compose.yaml (hunk 2 above)
Comment (CuriouzK0d3r) [reply docker-compose.yaml:513]: Yeah how will we inform the users?

Comment (vkotronis) [reply docker-compose.yaml:513]: Slack, the hard part is when to do this properly.

File: docker-compose.yaml (hunk 1 above)
Comment (CuriouzK0d3r) [reply docker-compose.yaml]: I replaced that and worked with /tmp this time :) can you check locally as well?

**Comment by vkotronis** [comment]
tests failing, turning to `draft` until issues resolved

**Comment by CuriouzK0d3r** [comment]
> tests failing, turning to `draft` until issues resolved

Tests are passing now :)

File: docker-compose.yaml (hunk 1 above)
Comment (vkotronis) [reply docker-compose.yaml:530]: could you replace with `/tmp/` then?

**Comment by vkotronis** [comment]
Please also add the new versions to the helm chart manifests. Also, could you fix the tests? I think there are linting errors, and then  the int tests should pass.

**Review by vkotronis — COMMENTED** [review]
Looks OK, just verify that `vagrant` also works (and that the new services are configured also there), by testing locally. I will have time to dig into the new images next week, and we can make a clean merge and inform the users they should upgrade their DBs. It will take some days to make sure that everything works OK, when you have cycles please test locally on an instance that is running for some time. You can use the FORTH demo configuration  file.

**Review by vkotronis — COMMENTED** [review]
Let's freeze this for now and we will update end of the month

**Review by vkotronis — CHANGES_REQUESTED** [review]
Please update all docker-compose files in root folder as well as artemis chart versions

**Pull request merged**